client/commands/v2/pysa_server.py

```diff
@@ -28,10 +29,14 @@
     _read_lsp_request,
     try_initialize,
     _log_lsp_event,
+    _publish_diagnostics,
     InitializationExit,
     InitializationSuccess,
     InitializationFailure,
 )
+from api import query, connection as api_connection
+from api.connection import PyreQueryError
```

Comment:
These need to be relative imports to work with our internal system

Reply:
I initially tried relative importing, but i get an`ImportError: attempted relative import beyond top-level package` for
```
from ....api import query, connection as api_connection
```

Reply:
hmm, I'm wondering how we distribute the `api` module. It feels like we should put it under a `pyre` module or something.